Mathlib/Algebra/CharP/ExpChar.lean

```diff
@@ -111,3 +112,79 @@ end NoZeroDivisors
 end Nontrivial
 
 end Semiring
+
+theorem ExpChar.exists [Ring R] [NoZeroDivisors R] [Nontrivial R] :
```

Comment:
Usually we write [IsDomain R] because these are equivalent in a Ring.

Reply:
I see. I just copied the conditions for `CharP.char_is_prime_of_pos` 🤣